src/main.cpp

```diff
@@ -1051,8 +1152,26 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
             return error("AcceptToMemoryPool: BUG! PLEASE REPORT THIS! ConnectInputs failed against MANDATORY but not STANDARD flags %s", hash.ToString());
         }
 
+        // Are we replacing an existing transaction?
+        if (pool.exists(hash))
+        {
+        }
```

Comment:
Lol, I must be blind... That's stub code replaced by the lines just below it; fixed, thanks!